.github/workflows/build.yml

```diff
@@ -3,7 +3,7 @@ name: build
 on:
   push:
     branches:
-      - '**'
```

Comment:
Don't build for every push, just on `master` so that every PR update doesn't trigger two builds